server.js

```diff
@@ -56,6 +59,13 @@ if (plugins.legacyGovukFrontendFixesNeeded()) {
 app.locals.pluginConfig = plugins.getAppConfig({
   scripts: utils.prototypeAppScripts
 })
+
+// Set up GOV.UK Frontend relative paths
+app.locals.govukFrontend = {
+  assetPath: govukFrontend.assetPath,
+  scripts: govukFrontend.config?.scripts || []
+}
```

Comment:
Just a note to say we can use `govukFrontendPaths()` to search elsewhere:

```js
utils.govukFrontendPaths({
  paths: ['/path/to/somewhere']
})
```

As I'm aware that both plugin pages and management pages use `assetPath`

Will we need to set different locals for each route? 

```js
require('./lib/manage-prototype-routes.js')
require('./lib/plugins/plugins-routes.js')
```